{{cookiecutter.project_slug}}/examples/slurm_cluster/run.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#SBATCH --account=rpp-bengioy
+#SBATCH --cpus-per-task=2
+#SBATCH --gres=gpu:1
+#SBATCH --mem=32G
+#SBATCH --time=5:00:00
+#SBATCH --job-name=__TODO__
+#SBATCH --output=logs/out_%a.log
+#SBATCH --error=logs/err_%a.log
+
+# 1. Create your environment
+module load python/3.7.4
+virtualenv --no-download $SLURM_TMPDIR/env
+source $SLURM_TMPDIR/env/bin/activate
+pip install --no-index --upgrade pip
+
+cp /__TODO__/path/to/requirements.txt $SLURM_TMPDIR/requirements.txt
```

Comment:
this menns we need to support requirements.txt - which makes sense -  but I think it's not the case now